internal/config/api/api.go

```diff
@@ -281,3 +277,11 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 
 	return cfg, nil
 }
+
+func GetCorsAllowOrigin(kvs config.KVS) []string {
+	corsAllowOrigin := strings.Split(env.Get(EnvAPICorsAllowOrigin, kvs.Get(apiCorsAllowOrigin)), ",")
```

Comment:
Should we make the condition as `if len(corsAllowOrigin) == 0 || (len(corsAllowOrigin) == 1 && corsAllowOrigin[0] == "")` for better readability?